.github/workflows/scripts/generate-weekly-report.js

```diff
@@ -114,6 +114,10 @@ async function getIssuesData({octokit, context}) {
       filterPrs: true,
       alias: "issuesSponsorNeeded",
     },
+    "waiting-for-code-owners": {
+      filterPrs: true,
```

Comment:
This label is valid for both PRs and issues, so I don't think we should filter PRs out of the report.

For some context, there was discussion on this label in the January 31, 2024 sig meeting. The original intention was to label PRs that were stuck because they hadn't received approval from code owners yet. We've then repurposed it as well where an issue is open that needs input from code owners for a final decision.